vllm/multimodal/utils.py

```diff
@@ -35,8 +35,8 @@ def _load_image_from_data_url(image_url: str):
     return load_image_from_base64(image_base64)
 
 
-def fetch_image(image_url: str) -> Image.Image:
-    """Load PIL image from a url or base64 encoded openai GPT4V format"""
+def fetch_image(image_url: str, *, image_mode: str = "RGB") -> Image.Image:
+    """Load PIL image from a HTTP or base64 data URL."""
```

Comment:
Let's still mention "OpenAI Vision format" here since the payload for base64 is something like
```
"url": f"data:image/jpeg;base64,{base64_image}"
```
instead of just the string itself.

Reply:
We should also call out in the docstring that we're converting the image to RGB

Reply:
Isn't that the same format as a standard data URL?

Reply:
> Isn't that the same format as a standard data URL?

Ah I misread your change and thought you meant just the string, yea we can just use Data URL as well (until OpenAI changes it)